src/services/api/restBuilder/restBuilder.ts

```diff
@@ -389,6 +389,7 @@ async function mutateParentCache(
           break;
         case "Update":
         // To do
+          break;
```

Comment:
This was an error that the linter detected.  Definitely don't want to do a delete on update, which fallthrough was allowing for.